gatsby/plugins/gatsby-source-cks/src/api/api.ts

```diff
@@ -0,0 +1,130 @@
+/**
+ * Low-level methods to access each of the CKS API endpoints
+ *
+ * @module
+ */
+
+import fetch from "node-fetch";
+import moment from "moment";
+import { Cache } from "gatsby";
+
+import {
+	ApiConfig,
+	ApiTopicsResponse,
+	ApiTopicResponse,
+	ApiTopicChangeResponse,
+	ApiPartialTopic,
+	ApiFullTopic,
+} from "./types";
+
+export const apiConfig: ApiConfig = {
+	apiBaseUrl: "",
+	apiKey: "",
+};
+
+export const configure = ({ apiBaseUrl, apiKey }: ApiConfig): void => {
+	apiConfig.apiBaseUrl = apiBaseUrl;
+	apiConfig.apiKey = apiKey;
+};
+
+const verifyConfig = (): true => {
```

Comment:
How come we return a value of true instead of bool, im not familiar with this? its interesting how you can not only specify the return type but also the return type value.

Reply:
Because it's TypeScript not JS. In this case, it doesn't return true or false, it returns true or throws so makes sense for the return type to be true